test/functional/activation_key/info_test.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/associating_commands'
+
+describe 'activation-key info' do
+  before do
+    @cmd = %w(activation-key info)
+  end
+
+  it "includes katello attributes" do
+    params = ['--id=1', '--organization-id=1']
```

Comment:
No need for `--organization-id` if you are using `--id`

Reply:
ah alright. I'll make that change. Thanks.